test/scheduler_test.rb

```diff
@@ -132,6 +132,19 @@
     assert job.opts.keys.include?(:first_in)
   end
 
+  test 'load_schedule_job with every fixnum' do
+    Resque::Scheduler.load_schedule_job(
+      'some_ivar_job',
+      'every' => 30,
+      'class' => 'SomeIvarJob',
+      'args' => '/tmp'
+    )
+
+    assert_equal(1, Resque::Scheduler.rufus_scheduler.jobs.size)
+    assert_equal(1, Resque::Scheduler.scheduled_jobs.size)
```

Comment:
Why not test the new job is well scheduled every 30s, and not 30m or 30ms ?

Reply:
agreed, we need to test the actual schedule that's configured in the scheduler here.

Reply:
@josephpage @iloveitaly definitely, updated!